concent-builder/files/jenkins/Jenkinsfile

```diff
@@ -0,0 +1,76 @@
+properties([[$class:
+'GithubProjectProperty',
+    displayName: '',
+    projectUrlStr: 'https://github.com/bartoszbetka/concent-deployment.git/'],
+pipelineTriggers([githubPush()])])
+
+stage ('Checkout repository') {
+    dir('/home/jenkins/concent-deployment/') {
+        git url:
+        'https://github.com/bartoszbetka/concent-deployment.git',
+        branch: 'CI'
+    }
+    ansiblePlaybook(
+        inventory: '/home/jenkins/concent-deployment-values/ansible_inventory',
+        playbook:  '/home/jenkins/concent-deployment/concent-builder/set-concent-version.yml',
```

Comment:
Please make this file a template so that we don't have to hard-code `/home/jenkins/`

Reply:
I don't remember exactly but I think in this file there is a problem with: `{` and `}` characters.